ci.jsonnet

```diff
@@ -243,6 +243,7 @@ local part_definitions = {
           "./native-image",
           "--no-server",
           "--language:ruby",
+          "--language:llvm",
```

Comment:
This should be `--language:llvm` first then `--language:ruby`, otherwise the configuration of llvm takes over ruby's.